subgraphs/curve-finance/src/common/utils.ts

```diff
@@ -450,9 +487,19 @@ export function getPoolFromGauge(gaugeAddress: Address): Address {
 }
 
 export function getPoolTVL(
-  outputToken: Token,
-  outputTokenBalance: BigInt
+  pool: LiquidityPool,
+  outputTokenBalance: BigInt,
+  block: ethereum.Block
 ): BigDecimal {
+  const outputToken = getOrCreateTokenFromString(pool.outputToken!, block);
+
+  if (outputToken.lastPriceUSD!.equals(constants.BIGDECIMAL_ZERO))
+    return getPoolTVLUsingInputTokens(
+      pool.inputTokens,
+      pool.inputTokenBalances,
+      block
+    );
+
   const totalValueLockedUSD = outputTokenBalance
     .divDecimal(
       constants.BIGINT_TEN.pow(outputToken.decimals as u8).toBigDecimal()
```

Comment:
use `exponentToBigDecimals()` function that is already written in place of this 

note: this is seen throughout